argocd/keycloak/values.yaml

```diff
@@ -1,3 +1,3 @@
----
+# ---
```

Comment:
Please don't comment this line

Reply:
Can you explain how having `---`  behaves differently from not having anything at all ?